train/arch-and-scaling-template.slurm

```diff
@@ -70,7 +70,6 @@ OPTIMIZER_ARGS=" \
     --lr 2e-4 \
     --min-lr 1e-5 \
     --lr-decay-style cosine \
-    --lr-decay-samples 73_242_187 \
```

Comment:
This should be set automatically IMO:
https://github.com/bigscience-workshop/Megatron-DeepSpeed/blob/2d9744f23df1a67b4cc1523e3bbdcaca738eb391/megatron/training.py#L337

This would prevent configs that are not exactly correct where the training sampels doesn't correspond to the lr_decay_samples anymore? 
https://github.com/bigscience-workshop/bigscience/blob/master/train/tr3-1B3-baseline/tr3m-1B3-pile.slurm

Reply:
Yes, agreed.